pages/linux/mycli.md

```diff
@@ -0,0 +1,15 @@
+# mycli
+
+> A CLI for MySQL, MariaDB, and Percona with auto-completion and syntax highlighting.
+
+- Connect to a database with current login user:
+
+`mycli {{database_name}}`
+
+- Connect to a database with specified user:
+
+`mycli -u {{user}} {{database_name}}`
+
+- Connect to a database with specified user and host:
```

Comment:
Perhaps `Connect to a database on the specified host with the specified user` would read better?

Reply:
How do you think
 `Connect to a database with the specified user and host` ?

Reply:
I'm not sure I understand the question.

Reply:
I think just one `specified` in the sentence is OK.
I'm not sure that `Connect to a database with the specified user and host` will be missunderstanding.

Reply:
Ahhh. Sure :-)